### PR TITLE
Add all Subsocial pallets to XSocial

### PR DESCRIPTION
**Pull request by F3Joule**

### CODE REVIEW

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -15,47 +15,56 @@ targets = ["x86_64-unknown-linux-gnu"]
...
+pallet-profiles = { default-features = false, git = "https://github.com/dappforce/subsocial-parachain.git", rev = "035c651a03ba94d78f1d5dbc87a5f4461cfbb664" }
+pallet-reactions = { default-features = false, git = "https://github.com/dappforce/subsocial-parachain.git", rev = "035c651a03ba94d78f1d5dbc87a5f4461cfbb664" }
 pallet-roles = { default-features = false, git = "https://github.com/dappforce/subsocial-parachain.git", rev = "035c651a03ba94d78f1d5dbc87a5f4461cfbb664" }
 pallet-space-follows = { default-features = false, git = "https://github.com/dappforce/subsocial-parachain.git", rev = "035c651a03ba94d78f1d5dbc87a5f4461cfbb664" }
-pallet-energy = { default-features = false, git = "https://github.com/dappforce/subsocial-parachain.git", rev = "035c651a03ba94d78f1d5dbc87a5f4461cfbb664" }
+pallet-space-ownership = { default-features = false, git = "https://github.com/dappforce/subsocial-parachain.git", rev = "035c651a03ba94d78f1d5dbc87a5f4461cfbb664" }
+pallet-spaces = { default-features = false, git = "https://github.com/dappforce/subsocial-parachain.git", rev = "035c651a03ba94d78f1d5dbc87a5f4461cfbb664" }
+# FIXME: change when merged to main
```
Comment (F3Joule) [inline runtime/Cargo.toml:63]:
```suggestion
# FIXME: change when merged to subsocial-parachain's main branch
```

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -90,12 +101,18 @@ std = [
...
+	"pallet-energy/std",
+	"pallet-permissions/std",
+	"pallet-posts/std",
+	"pallet-profiles/std",
+	"pallet-reactions/std",
+	"pallet-roles/std",
+	"pallet-space-follows/std",
+	"pallet-space-ownership/std",
+	"pallet-spaces/std",
+	"pallet-evm-accounts/std",
```
Comment (F3Joule) [inline runtime/Cargo.toml:115]: fix formatting

File: runtime/Cargo.toml [hunk 3]
```diff
@@ -104,8 +121,20 @@ runtime-benchmarks = [
...
+	"pallet-domains/runtime-benchmarks",
+	"pallet-energy/runtime-benchmarks",
+	"pallet-reactions/runtime-benchmarks",
+	"pallet-roles/runtime-benchmarks",
+	"pallet-space-follows/runtime-benchmarks",
+	"pallet-space-ownership/runtime-benchmarks",
+	"pallet-spaces/runtime-benchmarks",
+	"pallet-posts/runtime-benchmarks",
+	"pallet-profiles/runtime-benchmarks",
+	"pallet-evm-accounts/runtime-benchmarks",
```
Comment (F3Joule) [inline runtime/Cargo.toml:137]: fix formatting

File: runtime/Cargo.toml [hunk 4]
```diff
@@ -125,4 +156,16 @@ try-runtime = [
...
+	"pallet-energy/try-runtime",
+	"pallet-permissions/try-runtime",
+	"pallet-posts/try-runtime",
+	"pallet-profiles/try-runtime",
+	"pallet-reactions/try-runtime",
+	"pallet-roles/try-runtime",
+	"pallet-space-follows/try-runtime",
+	"pallet-space-ownership/try-runtime",
+	"pallet-spaces/try-runtime",
+	"pallet-evm-accounts/try-runtime",
```
Comment (F3Joule) [inline runtime/Cargo.toml:170]: fix formatting

File: runtime/src/lib.rs [hunk 5]
```diff
@@ -114,10 +115,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("xsocial"),
 	impl_name: create_runtime_str!("subsocial-xsocial-testnet"),
 	authoring_version: 1,
-	spec_version: 100,
+	spec_version: 102,
```
Comment (F3Joule) [inline runtime/src/lib.rs:118]: 101 was adding Energy pallet, merged with #2 and `spec_version` 100

File: runtime/src/lib.rs [hunk 6]
```diff
@@ -114,10 +115,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
...
 	impl_version: 1,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 1,
+	transaction_version: 2,
```
Comment (F3Joule) [inline runtime/src/lib.rs:121]: Pallet indexes changed.

**Review by olehmell — APPROVED** [review]

**Review by olehmell — APPROVED** [review]

**Pull request merged**